src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -118,18 +123,31 @@ public CallbackService(final SecurityUtils securityUtils,
         HttpHeaders securityHeaders = securityUtils.authorizationHeaders();
 
         CallbackTelemetryThreadContext.setTelemetryContext(new CallbackTelemetryContext(callbackType));
+        List<String> ccdCallbackLogControl = applicationParams.getCcdCallbackLogControl();
+        boolean match = false;
+        String starStr = "*";//to match any call back
+        if (ccdCallbackLogControl.size() > 0 && (starStr.equals(ccdCallbackLogControl.get(0))
+            || ccdCallbackLogControl.stream().anyMatch(url::contains))) {
+            match = true;
+        }
```

Comment:
we could do an extract method here to simplify the code. 
All this could go into a private method
`boolean logCallbackDetails(....)`
and then it can be directly called on line 148: e.g.
```if(logCallbackDetails(...) {
     ....
```